cookiecutter.json

```diff
@@ -46,6 +46,10 @@
     "blue",
     "black"
   ],
+  "code_security": [
```

Comment:
as we could have more options for code security running at the same time ... I think that it doesn't make sense to have it as a list of options. it would be better to just check if the user want to install bandit or not
maybe, something like
```
  "use_bandit": 'y',
```

Reply:
Can the conditionals for that case be {%- if cookiecutter.use_bandit in ["yes", "y"] %} or just {%- if cookiecutter.use_bandit == "y" %}?

Reply:
not sure, TBH, you need to check that .. also, not sure if it will be converted automatically to `True`